README.md

```diff
@@ -41,6 +41,9 @@ Feel free to raise issues to ask questions or report bugs.
 ## Contributing
 
 Comments and suggestions for improvements are most welcome.
+
+You are an owner of an ABAP object type? We will be more than happy if you contribute your object type to this project. Just open a new pull request. 
```

Comment:
```suggestion
You feel responsible for an ABAP object type? We will be more than happy if you contribute your object type to this project. Just open a new pull request. 
```

Reply:
Yes, I see your point, but I think we have to involve the object owners. At least they have to review it. Should we add this somewhere?

Reply:
Sure, but getting PRs and reviewing them are 2 different issues. Here, we primarily want to invite others, including object owners and external contributors, to collaborate. It goes without saying that any PR will be reviewed by us as maintainers and other experts as required. But I don't see a need to stress this point here.